os/se/ameba/rtl_security_api_wrapper.c

```diff
@@ -239,6 +239,37 @@ uint32_t rtl_read_factory_cert_wrapper(hal_data *data)
 	return ret;
 }
 
+uint32_t rtl_write_factory_cert_wrapper(hal_data *data)
+{
+	uint32_t ret;
+	uint8_t buf_length[4];
+	flash_t flash;
+	uint32_t data_len = data->data_len;
+
+	uint8_t	*buf_flash = (uint8_t *) malloc(data_len + 4);
```

Comment:
Please remove space after type casting. 
And please use kmm_malloc instead of malloc, because this module is in kernel side.

Reply:
Modified.